src/pages/Admin/Room/Modal/Item/Item.tsx

```diff
@@ -63,6 +64,7 @@ const RoomModalItem: React.FC<RoomModalItemProps> = ({
       const valuesWithTemplate = {
         ...values,
         template,
+        isEnabled: true,
```

Comment:
I'm assuming this is the part I called out in https://github.com/sparkletown/sparkle/pull/1176#issuecomment-779632114 that required further checking?

Reply:
Correct.

Reply:
Cool. Probably worth leaving this open until we have a resolution one way or the other.

Reply:
https://github.com/sparkletown/sparkle/pull/1176#issuecomment-779703122